containers/test-apps/kitchen/bin/kitchen

```diff
@@ -943,10 +948,22 @@ class Kitchen(HTTPWebSocketsHandler):
                  'request-length': self.__request_body_length,
                  'request-method': self.__method,
                  'uri': self.__path }
+
         if len(self.headers) > 0:
             info['headers'] = { k.lower(): v for k, v in self.headers.items() }
+
+            cookie_value = info['headers'].get('authorization', None)
+            if cookie_value is not None:
+                info['headers']['authorization'] = truncate(cookie_value, 15)
+
+            cookie_value = info['headers'].get('cookie', None)
```

Comment:
The explicit `None` here is redundant:
https://docs.python.org/3/library/stdtypes.html#dict.get

Reply:
Done, remove `None`.